### DISCUSSION TITLE
Request Approved a week ago yet no benefits

### DISCUSSION THREAD

**Discussion by AnjaneyMitra** [community/community#134918] [GitHub Education]
### Hi everyone,

I submitted my request on 1st august and it was approved, yet it still shows that my benefits will be available soon. 
What might be the issue?

**Comment by davevad93** [comment]
Hi @AnjaneyMitra ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AnjaneyMitra was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by Chaitanya029** [comment]
Hi @AnjaneyMitra,

It's possible that there's a delay in updating the status of your benefits. I've seen cases where it takes a few days for the benefits to become available after approval. For example, someone who was approved on June 11, 2023, had to wait for 4 days for their benefits to become available.

Additionally, GitHub has recently updated its application and re-verification process, which might be causing some delays.

If you haven't received an email notification about your benefits, you might want to check your email inbox and spam folder. Some users have reported not receiving an email notification after being approved for student benefits.

If the issue persists, you might want to reach out to GitHub support for further assistance.
